views/components/footer.njk

```diff
@@ -1,12 +1,28 @@
-<footer class="d-flex fixed-bottom flex-wrap justify-content-between align-items-center py-3 border-top" style="background-color:white">
-    <p class="col-md-4 mb-0">© 2024 Kainos</p>
-    
-    <div class="nav-logo-container">
-        <img src="../assets/index/kainos-logo.png" alt="transparent kainos logo for the navbar" class="img-fluid">
-    </div>
+{% if theme == 'halloween' %}
```

Comment:
You have three footers, a halloween one, a christmas one and a normal one. You should split this out into multiple files and componentize it.

Reply:
Done